.github/workflows/vrf.contracts.test.yaml

```
@@ -0,0 +1,55 @@
+name: "contracts: test+publish"
```

I would suggest to change the name for easier orientation in tests displayed in GitHub. And I think we should not publish using this github action.

```suggestion
name: "vrf contracts: test"
```